docs/src/main/asciidoc/security.adoc

```diff
@@ -275,4 +275,95 @@ If a Quarkus tag represented by the given CPE name entry is affected by some CVE
 We will be asking the NVD CPE team to update the list as well as link Quarkus CPE name entries with the related CVEs on a regular basis.
 If you work with the link:https://jeremylong.github.io/DependencyCheck/dependency-check-maven/[OWASP Dependency Check Plugin] which is using NVD feeds to detect the vulnerabilities at the application build time and see a false positive reported then please re-open link:https://github.com/quarkusio/quarkus/issues/2611[this issue] and provide the details.
 
-Note link:https://jeremylong.github.io/DependencyCheck/dependency-check-maven/[OWASP Dependency Check Plugin] `6.2.0` or later should be used with Quarkus.
+You can add `OWASP Dependency Check Plugin` to your project's `pom.xml` like this:
+
+[source,xml]
+----
+<plugin>
+    <groupId>org.owasp</groupId>
+    <artifactId>dependency-check-maven</artifactId>
+    <version>${owasp-dependecy-check-plugin.version}</version>
+    <configuration>
+        <!-- Fail only when detecting High Vulnerability issues -->
+        <failBuildOnCVSS>7</failBuildOnCVSS>
+        <suppressionFiles>
+            <suppressionFile>${project.basedir}/dependency-cpe-suppression.xml</suppressionFile>
+        </suppressionFiles>
+    </configuration>
+</plugin>
+---- 
+
+You can change `failBuildOnCVSS` value to detect less severe issues as well.
+
+A suppression list may wary depending on whether you'd like to keep checking the false positives to avoid missing something or not.
```

Comment:
```suggestion
A suppression list may vary depending on whether you'd like to keep checking the false positives to avoid missing something or not.
```